server/agent/src/main/java/ai/starwhale/mlops/agent/task/persistence/FileSystemTaskPersistence.java

```diff
@@ -34,104 +36,212 @@
 @Service
 public class FileSystemTaskPersistence implements TaskPersistence {
 
-    private final String infoSuffix = ".taskinfo";
-    private final String statusSuffix = ".status";
+    String baseTaskPathFormat = "%s/%s/";
+
+    String infoSuffix = "taskInfo.json";
+
+    String infoFilePath = baseTaskPathFormat + infoSuffix;
+
+
+    String statusFilePath = baseTaskPathFormat + "status.txt";
```

Comment:
some file name in container advice:
- simple status file:  /var/starwhale/status/current
- full timeline status file:  /var/starwhale/status/timeline

plain text remove `.txt` postfix and use explict filename?